tests/runtime/call

```diff
@@ -661,13 +661,13 @@ PROG BEGIN { if (*percpu_kaddr("process_counts") == *percpu_kaddr("process_count
 EXPECT SUCCESS
 
 NAME percpu_kaddr field access
-PROG BEGIN { $runq = ((struct rq *)percpu_kaddr("runqueues", 0)); if ($runq != 0) { printf("nr_running: %d\n", $runq->nr_running) } exit() }
-EXPECT_REGEX nr_running: [0-9]+
+PROG BEGIN { $runq = ((struct rq *)percpu_kaddr("runqueues", 0)); if ($runq != 0) { printf("nr_queued: %d\n", $runq->nr_queued) } exit() }
```

Comment:
GHA runners certainly don't have kernels with this change at the moment. Ideally, we should change this test to use some different field of `runqueues` which is stable across kernels.

Reply:
My bad, I confused `struct cfs_rq` with `struct rq` (my lack of scheduler knowledge is evident here), the latter do still have the `nr_running`.

Those can be left as-is.